src/components/molecules/MageInformation/index.tsx

```diff
@@ -25,11 +25,23 @@ type Props = {
   theme: any
 }
 
+function formatCharges(
+  numberOfCharges: number | 'Custom',
```

Comment:
Ah, I didn't realize that the type is more complex - we should probably extract a type `NumberOfCharges` in the future and use that instead of copying this around. But for now this will do :)